components/controls/Modal.css

```diff
@@ -101,6 +101,10 @@
   line-height: normal;
 }
 
+.modal-head h2 {
+  overflow-wrap: break-word;
+}
+
```

Comment:
At least this is quick and simple fix for the current issue at hand!
I think you should even merge it to the previous class so that is apply to both `.modal-head h1, .modal-head h2`...